src/NetAnalyzers/Microsoft.CodeAnalysis.NetAnalyzers.sarif

```diff
@@ -153,7 +154,7 @@
           "id": "CA1507",
           "shortDescription": "Use nameof to express symbol names",
           "fullDescription": "Using nameof helps keep your code valid when refactoring.",
-          "defaultLevel": "warning",
+          "defaultLevel": "note",
```

Comment:
I plan to change "note" to "suggestion" and "hidden" to "silent" in the sarif generation tool as a separate PR.